docs/api/field_types_reference/measurementfield.md

```diff
@@ -0,0 +1,135 @@
+# Measurement Field Type
+
+The Measurement Field Type represents measurement information. 
+It stores the unit of measure, and either a single measurement value, 
+or a pair of values that defines a range.
+
+| Name          | Internal name       | Expected input type                                |
+|---------------|---------------------|----------------------------------------------------|
+| `Measurement` | `ibexa_measurement` | `Ibexa\Contracts\Measurement\Value\ValueInterface` |
+
+## PHP API Field Type
+
+### Input expectations
+
+To create a value, a Service of `Ibexa\Contracts\Measurement\MeasurementServiceInterface` 
+type is required.
```

Comment:
```suggestion
To create a value, a service implementing `Ibexa\Contracts\Measurement\MeasurementServiceInterface` 
is required.
```
not sure about this, but I don't think we talk about "being a type of interface"?